controls/3.5.rb

```diff
@@ -164,9 +164,9 @@
   end
   keyring_locations.uniq!.sort!
 
-  describe "[#{gcp_project_id}] KMS Regions" do
+  describe "[#{gcp_project_id}] KMS Regions #{ keyring_locations.sort }" do
     subject { keyring_locations }
-    it { should cmp kms_regions_list.sort }
+    it { should be_in kms_regions_list.sort }
```

Comment:
No need to sort here unless you just want it more readable but yes I think this will work ...

Reply:
keeping sorts for humans